### PR TITLE
Add rules `selector-attribute-operator-whitelist` and `selector-attribute-operator-blacklist`.

### PR DESCRIPTION
**Pull request by evilebottnawi**
/cc @davidtheclark @jeddy3

### CODE REVIEW

**Comment by jeddy3** [comment]
Looks great!

If we agree that regex support isn't needed, then happy to see this merged.

**Comment by evilebottnawi** [comment]
@jeddy3 The number of operators is not much so I think regexp support is not necessary if in the future we will get requests for it, there will be no problem to add this. :smile:

File: docs/user-guide/rules.md [hunk 1]
```diff
@@ -135,8 +135,10 @@ Here are all the rules within stylelint, grouped by the [*thing*](http://apps.wo
 ### Selector
 
 - [`selector-attribute-brackets-space-inside`](../../src/rules/selector-attribute-brackets-space-inside/README.md): Require a single space or disallow whitespace on the inside of the brackets within attribute selectors.
+- [`selector-attribute-operator-blacklist`](../../src/rules/selector-attribute-operator-blacklist/README.md): Specify a blacklist of disallowed operators.
 - [`selector-attribute-operator-space-after`](../../src/rules/selector-attribute-operator-space-after/README.md): Require a single space or disallow whitespace after operators within attribute selectors.
 - [`selector-attribute-operator-space-before`](../../src/rules/selector-attribute-operator-space-before/README.md): Require a single space or disallow whitespace before operators within attribute selectors.
+- [`selector-attribute-operator-whitelist`](../../src/rules/selector-attribute-operator-whitelist/README.md): Specify a whitelist of allowed operators.
```
Comment (davidtheclark) [inline docs/user-guide/rules.md]: Let's make this clear everywhere that they were "attribute operators", not just operators.

File: src/rules/selector-attribute-operator-whitelist/README.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+ * These operators */
+```
+
+## Options
+
+`array`: `"["array", "of", "operators"]`
+
+### `["array", "of", "operators"]`
+
+Whitelisted properties are the only *allowed* properties.
```
Comment (davidtheclark) [inline src/rules/selector-attribute-operator-whitelist/README.md]: operators, not properties.

**Comment by davidtheclark** [comment]
> The number of operators is not much so I think regexp support is not necessary

Agreed.

I just asked for a couple of sentence cleanups.

**Comment by davidtheclark** [comment]
Great! Let's merge after green light.

**Comment by evilebottnawi** [comment]
CI green. Merge.

**Pull request merged**